molecule/shared/substrate/lxd/create.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Manage LXD containers
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    molecule_inventory:
+      molecule:
+        hosts: {}
+    molecule_yml:
+      platforms:
+        - name: instance
+          image: ubuntu:20.04
+          user: ansible
```

Comment:
If molecule is setup correctly this should be pulled in from

https://github.com/deeep-network/bedrock/blob/4f423edd99283be38b6f3f63625086a29009487d/molecule/default/molecule.yml#L6-L8

when using a command like `molecule converge` from the root directory

Reply:
changed to noble